### PR TITLE
[Setup] Allow a set of image types (or all) for ImageUpload

### PR DESCRIPTION
**Pull request by StefanoA1**
**Detailed purpose of the PR**

- Allow a set of image types (or all) for ImageUpload

**Result and observation**

Kapture 2021-03-31 at 19 54 48

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/atom/image-upload/index.js [hunk 1]
```diff
@@ -14,7 +16,7 @@ const ImageUpload = ({
   onChange,
   onReset = null,
   name,
-  imageType = '*'
+  imageTypes
```
Comment (godu) [inline packages/@coorpacademy-components/src/atom/image-upload/index.js:19]:
```suggestion
  imageTypes = ['*']
```

File: packages/@coorpacademy-components/src/atom/image-upload/index.js [hunk 2]
```diff
@@ -25,6 +27,7 @@ const ImageUpload = ({
         [onReset]
       )
     : null;
+  const acceptedImages = imageTypes ? join(',image/', imageTypes) : '*';
```
Comment (godu) [inline packages/@coorpacademy-components/src/atom/image-upload/index.js:30]:
```suggestion
  const acceptedImages = pipe(map(t => `image/${t}`), join(','))(imageTypes);
```

File: packages/@coorpacademy-components/src/atom/image-upload/index.js [hunk 3]
```diff
@@ -41,7 +44,7 @@ const ImageUpload = ({
           <input
             type="file"
             name={name}
-            accept={`image/${imageType}`}
+            accept={`image/${acceptedImages}`}
```
Comment (godu) [inline packages/@coorpacademy-components/src/atom/image-upload/index.js:47]:
```suggestion
            accept={acceptedImages}
```

**Review by godu — CHANGES_REQUESTED** [review]

**Review by DjamelSoualmi — APPROVED** [review]

**Comment by StefanoA1** [comment]
[setup: add input (pass watermark)](https://trello.com/c/1mcbWSgS/115-setup-add-input-pass-watermark)

**Pull request merged**